common-lib/src/cmr/common/test/runners/ltest.clj

```diff
@@ -0,0 +1,38 @@
+(ns cmr.common.test.runners.ltest
+  "This test runner is a CMR adaptor for the ltest Clojure test runner."
+  (:require
+   [clojure.stacktrace :as stack]
+   [clojure.string :as string]
+   [clojure.test :as test]
+   [cmr.common.test.runners.util :as util]
+   [ltest.core :as ltest]
+   [potemkin :refer [import-vars]]))
+
+(import-vars
```

Comment:
I took a look at the potemkin documentation and wasn't entirely clear on what the import-vars macro does for us. Could you explain?